index.js

```diff
@@ -48,6 +49,11 @@ module.exports = opts => {
 	});
 
 	app.on('ready', () => {
+		// activate devtron for the user if they have it installed
+		try {
```

Comment:
I don't see any reason the user wouldn't want it if they have it installed. It's totally optional to go to that tab too.